ui/admin/settings.php

```diff
@@ -1,5 +1,5 @@
 <div class="wrap pods-admin">
-	<div class="pods-admin__content-container">
+	<div class="pods-admin-container">
```

Comment:
The `__` is BEM I believe https://en.bem.info/methodology/naming-convention/ and I see that used more and more alongside newer React/JS projects. I think this might be fine as it was.

Reply:
Ah, that explains!

Reply:
Maybe PodsUI should be updated as well then because it still uses `pods-admin-container`.
I'm all for using a naming convention, but if we do we should update everything.. not just the newly added parts because that creates a mess..